.travis.yml

```diff
@@ -1,7 +1,10 @@
-language: java
-
-jdk:
- - oraclejdk7
-
-script:
-  ./ci/lint.bash
+script: "./ci/lint.bash"
+before_deploy: pip install --user awscli
+deploy:
+- provider: script
+  skip_cleanup: true
+  script: "./ci/push.bash"
+  on:
+    tags: true
+    branch: master
```

Comment:
I still don't think we need `branch: master` here, because actual deploy will happen only on tags.

Reply:
Definitely.